src/styles/stars.scss

```
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  margin: 0;
+  &__star {
```

```suggestion
  margin: 0;

  &__star {
```
fix all similar cases - add blank line before each selector